contrib/initramfs/hooks/zfsunlock.in

```diff
@@ -0,0 +1,65 @@
+#!/bin/sh
+
+PREREQ="dropbear"
+
+prereqs() {
+    echo "$PREREQ"
+}
+
+case "$1" in
+    prereqs)
+        prereqs
+        exit 0
+    ;;
+esac
+
+. /usr/share/initramfs-tools/hook-functions
+
+if [ -r "${DESTDIR}/sbin/dropbear" ]; then
+	cat >"$DESTDIR"/bin/zfsunlock <<'EOF'
```

Comment:
Rather than having the `zfsunlock` initramfs hook script write out `/bin/zfsunlock` with `cat` and a HERE doc, could we just install what is the `/bin/zfsunlock` script somewhere on the system (whether that's in `/bin`, or `/usr/share/initramfs-tools/...` is a separate question) and then `cp` (or `install` if you want to merge the `cp` and `chmod`) it into the initrd?